public/front-channel.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Deriv</title>
+        <meta charset="utf-8" />
+        <script>
+            try {
+                // we check if we can access the top.location, if we can't it will throw an error
+                // and this means this front-channel is not rendered on the same window as the application which is logging out
+                let isSameOrigin = top.location.hostname === self.location.hostname;
```

Comment:
`isSameOrigin` are we reusing this variable again ?

Reply:
No this is not being used anywhere, its just for readability purposes
We just check this to make sure we don't invoke front channels to clear the local storage on the same page (i.e. DBot)